cla_public/templates/_phase-tag.html

```diff
@@ -10,10 +10,25 @@
       <a lang="cy" href="{{ url_for('base.set_locale', locale='cy_GB') }}" data-ga="event:locale/switch/welsh">Cymraeg</a>
     {% endif %}
   </div>
-
+  {#
```

Comment:
Could we remove the original block altogether? We can revert this PR to restore it if we need to.